### DISCUSSION TITLE
How to use R console when on a remote server?

### DISCUSSION THREAD

**Discussion by jarbet** [posit-dev/positron#5183] [Q&A]
Hello,

The R console works great locally on my Mac.  However, when I use the Remote Explorer tab to ssh onto a remote linux server, the console no longer works (it is stuck on "Starting up...").

On my linux server, I use R through a conda environement.  I can activate my R environment in the terminal, start R in the terminal, but then the plotting and help page functionalities no longer work (e.g. `plot(1)` will not open a plot in the far right pane).

Any advice on how to fix this?  Is there some setting in Positron where I can tell it where my R executable path is on the linux server, and link that to the Console somehow?

**EDIT:** My linux server uses  AlmaLinux 8.7 (Stone Smilodon)

**Comment by jmcphers** [comment]
Can you share your R logs so we can investigate further? https://positron.posit.co/troubleshooting.html#python-and-r-logs

It is not currently possible to specify where R is located (Positron just scans likely candidates), but we do plan to add one. Feel free to comment/upvote this issue: https://github.com/posit-dev/positron/issues/2235
